lib/spatial_features/download.rb

```diff
@@ -26,6 +26,12 @@ def self.normalize_file(file)
       end
     end
 
+    def self.entries(file)
```

Comment:
This name is kind of weird but it keeps things consistent with the rest of the lib so I just went with it.